services-api/src/main/java/io/scalecube/services/ServiceCall.java

```diff
@@ -364,31 +375,34 @@ public ServiceCall requestReleaser(Consumer<Object> requestReleaser) {
                 return check.get(); // toString, hashCode was invoked.
               }
 
+              Integer requestParameterIndex = Reflect.requestParameterIndex(method);
```

Comment:
please use `OptionalInt`

Reply:
This part of the code doesn't exist anymore.